File: .old/v4/engine/ai/cardealer.lua
local CarDealer = setmetatable({}, ai.NPC)
CarDealer.__index = CarDealer

function CarDealer:new(x, y)
   return ai.NPC.new(self, x, y)
end

-------------
-- PRIVATE --
-------------

function CarDealer:__tostring()
   return '[CarDealer]'
end

function CarDealer:type() return 'CarDealer' end

return CarDealer

-- vim: ts=3:sw=3:sts=3:expandtab

File: engine/npc/cardealer.cc
#include "engine/npc/cardealer.h"

#include <memory>
using namespace std;

#include "./main.h"
#include "ui/ui.h"
#include "ui/dialogmanager.h"

CarDealer::CarDealer(Point init, class City& city)
    : Worker(init, city, 0)
{
}

void
CarDealer::TalkToHero()
{
}


// vim: ts=4:sw=4:sts=4:expandtab
